corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -248,6 +248,13 @@ FormplayerFrontend.module("Menus.Views", function (Views, FormplayerFrontend, Ba
         },
     });
 
+    Views.PersistentCaseTileView = Views.CaseTileView.extend({
+        rowClick: function (e) {
+            e.preventDefault();
+            FormplayerFrontend.trigger("menu:show:detail", this.options.model.get('id'), 0, true);
```

Comment:
nit i don't think you actually need to use `this.options`, backbone i believe saves it as `this.model`